cmd/node/node.go

```diff
@@ -445,10 +446,8 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeID,
 
 	app.nodeID = nodeID
 
-	name := nodeID.ShortString()
-
-	lg := log.NewDefault(name).WithFields(nodeID)
-
+	// This base logger must be debug level so that other, derived loggers are not a lower level.
+	lg := log.NewWithLevel(nodeID.ShortString(), zap.NewAtomicLevelAt(zapcore.DebugLevel)).WithFields(nodeID)
```

Comment:
I think this is the only change needed - to have the default logger run with debug level.

Reply:
I tested this (https://github.com/spacemeshos/go-spacemesh/pull/2071/commits/d1c68d28339c8da7070d5ed066fe71f917afa7c3), it didn't work. The issue is deeper than this. I explained why in #2090. I don't think we can make a change this superficial here.